cicd/pipeline.yaml

```diff
@@ -264,7 +266,9 @@ Resources:
             Configuration:
               ProjectName: !Ref EmptyBucket
               EnvironmentVariables: '[
-                                       { "name":"DOCUMENTATION_BUCKET", "value":"#{ExportConfigNS.DOCUMENTATION_BUCKET}" }
+                                       { "name":"DOCUMENTATION_BUCKET", "value":"#{ExportConfigNS.DOCUMENTATION_BUCKET}" },
+                                       { "name":"BUCKET_PATH", "value":"#{ExportConfigNS.BUCKET_PATH}" },
+                                       { "name":"KEEP_API_SPECS", "value":"false" }
```

Comment:
Hi Joe, Why KEEP_API_SPECS is false here? I don't think we should delete api specs at any stage. Because those api specs are required to render slate API UI.

Reply:
It's only false here when this action runs in the teardown stage. The bucket can't be removed unless it's completely empty. Teardown only gets run in non production environments when we want to clean everything up. 

Specs are kept here as part of the deploy https://github.com/AtlasOfLivingAustralia/ala-api-portal/blob/feature-CF-templates/cicd/pipeline.yaml#L213

Reply:
ah ok, Thanks Joe